### ISSUE TITLE
Better inline description of what this group is about

### ISSUE THREAD

**Issue by wlach** [mozilla-rpweb/discussions#5]
@eliperelman's initial mail to an internal list was great (there's nothing confidential in there so I'll just copy it here), but it's not obvious to people first coming to this github site what this thing is about. Do you think we could put something in the README.md describing this project's purpose and intent?

```
Hello R&P!

Over the past few months, a few of us that work on our frontend projects started noticing some issues with our efforts as they relate to being on different teams. As we have started collaborating more across our teams, we have come to the conclusion that we could be more efficient and helpful to others if we organized ourselves a little better and work together to solve common problems.

- How can we quickly start and iterate on new web projects?
- What practices should we adopt across our projects?
- How can we provide familiar and aesthetic user experiences for our apps?

I'm happy to announce that we have formed a cross-functional group of people dedicated to web and front-end to tackle these kinds of issues, and to provide our assistance to everyone in the org where we can in implementing. Our goal is to answer the above questions and more, so we can quickly and efficiently work together for releasing apps, tooling, guides, and documentation for these kinds of projects.

And we want to help you!

If you have web and front-end needs, we have started a GitHub org to track our projects and discussions for your assistance. If you need help starting a new project, need advice on crating better UIs, or anything related to a web project, you can come to us. If we can't dedicate our time to writing something for you, we will assist you in at least getting a push in the right direction.

File an issue at https://github.com/mozilla-rpweb/discussions if you want to discuss something! If you also do a majority of your work on the front end, we would love for you to get involved.

Thanks again, and hope to help you all in the future!

Eli Perelman, Hassan Ali, David Walsh, Cameron Dawson, and Israel Madueme
Release & Productivity Web
https://github.com/mozilla-rpweb/discussions
```

**Comment by eliperelman** [comment]
I think this is a great idea.

**Comment by eliperelman** [comment]
Fixed in cafe7813b30ffecc6455c7e670261ca7ffa949ce.
